shell/common/crash_keys.cc

```diff
@@ -186,8 +186,10 @@ void SetCrashKeyForGinWrappable(gin::WrapperInfo* info) {
     crash_location = "Notification";
   else if (info == &electron::api::Cookies::kWrapperInfo)
     crash_location = "Cookies";
+#if BUILDFLAG(ENABLE_DESKTOP_CAPTURER)
```

Comment:
Can you explicitly add `#include "electron/buildflags/buildflags.h"` for this declaration.

Reply:
done